src/screens/welcome/Welcome.tsx

```diff
@@ -30,6 +30,7 @@ export const Welcome = ({ route }: AuthNavigationProps<'WELCOME'>) => {
   const { updateUser } = useUserContext()
   const { mutate: createTempUser } = useCreateTempUser()
   const { notify } = useGetNotificationsConfig()
+  const [isLoading, { setFalse: setIsNotLoading, setTrue: setIsLoading }] = useBooleanState(false)
```

Comment:
```suggestion
  const [isLoading, { setFalse: showLoader, setTrue: hideLoader }] = useBooleanState(false)
```
nothing wrong with your naming convention, this just feels more explicit to me. WDYT?